src/js/components/Pagination/Pagination.js

```diff
@@ -177,6 +177,10 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
+    useEffect(() => {
```

Comment:
For readability, I'm wondering if we should move this up to line 44 so it is next to where `activePage` is created.

Reply:
My reasoning to move them at the end was to easily extend it latter if needed, and add dependencies declared above.
But I agree it makes more sense to have ti close to `activePage`